docs/api/plugins/browser-launch-api.mdx

```diff
@@ -383,6 +383,31 @@ on('before:browser:launch', (browser, options) => {
 
 :::
 
+### Disable `--headless=new` for Chrome
+
+Starting in Chrome version 112, Cypress now will now launch Chrome with the with
```

Comment:
Do we need to specify what version of cypress we introduced this in since we aren't back patching this?

Reply:
```suggestion
Starting in Cypress 12.15.0 for Chrome version 112, Cypress now will now launch Chrome with the with
```